integrations/maya/src/main/java/org/dash/wallet/integrations/maya/utils/MayaConstants.kt

```diff
@@ -17,7 +17,19 @@
 
 package org.dash.wallet.integrations.maya.utils
 
+import org.bitcoinj.core.NetworkParameters
+
 object MayaConstants {
+    const val DEFAULT_EXCHANGE_CURRENCY = "USD"
```

Comment:
The default currency of Maya is USD.  This cannot be changed, but we can covert rates to other currencies with an external source.